src/RoslynInsertionTool/RoslynInsertionTool.Commandline/Settings.settings

```diff
@@ -71,5 +71,8 @@
     <Setting Name="VisualStudioRepoAzdoUri" Type="System.String" Scope="Application">
       <Value Profile="(Default)">https://devdiv.visualstudio.com/DefaultCollection</Value>
     </Setting>
+    <Setting Name="SkipCoreXTPackages" Type="System.String" Scope="Application">
+      <Value Profile="(Default)">VS.Tools.Roslyn</Value>
```

Comment:
I don't think we should have a default value for this option. Here or in the app.config.

Reply:
changed